server/routes/api/stats.py

```diff
@@ -414,65 +414,212 @@ def get_place_stat_date_within_place():
     return Response(json.dumps(response), 200, mimetype='application/json')
 
 
-def get_set_csv_rows(stat_set, sv_list, row_limit=None):
+def get_facets_by_variable_series(series_response):
```

Comment:
maybe rename to: get_variable_facets_from_series()

Reply:
done